src/datajudge/constraints/uniques.py

```diff
@@ -292,3 +294,76 @@ def compare(
     def test(self, engine: sa.engine.Engine) -> TestResult:
         self.max_relative_gain = self.max_relative_gain_getter(engine)
         return super().test(engine)
+
+
+class VariantDistributionConstraint(Constraint):
+    """
+    VariantDistributionConstraint is a class to check if the distribution of values in a column
+    falls within the specified minimum and maximum bounds.
+    The constraint compares the factual distribution of values in a column of a `DataSource`
+    with the target distribution supplied as a dictionary where keys represent the unique values
```

Comment:
Nitpick: I think the keys represent some unique values, not necessarily all of them, right?